src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -129,6 +129,13 @@ public void clear() {
         internalList.clear();
     }
 
+    /**
+     * Returns size of address book
+     */
+    public int size() {
```

Comment:
Names representing methods must be verbs and written in camelCase. How about `getSize()` instead?